students/wielgus_daniel/lesson_01_basics/lesson_01_part_02.py

```diff
@@ -0,0 +1,124 @@
+import math
+
+#Problem1: Last digit of integer
+print("We will show Last digit of integer")
+print("Please provide the digit:")
+a = int(input())
+b = str(a)
+print("The last digit is: " + b[-1])
+print("THE END of Last digit of integer problem")
+print("")
+
+#Problem2: Tens digit
+print("We will show Tens digit")
+print("Please provide the digit:")
+a = int(input())
+b = str(a)
+print("The Tens digit is: " + b[-2])
+print("THE END of Tens digit problem")
+print("")
+
+#Problem3: Sum of digits
+print("We will show sum of digit")
+print("Please provide the three-digit number:")
+number = int(input())
+strNumber = str(number)
+firstDigit = strNumber[-3]
+secondDigit = strNumber[-2]
+thirdDigit = strNumber[-1]
+sumOfDigits = int(firstDigit) + int(secondDigit) + int(thirdDigit)
+print("The sum of digits is: " + str(sumOfDigits))
+print("THE END of Sum of digit problem")
+print("")
+
+#Problem4: Fractional part
+print("We will show Fractional part")
+print("Please provide the positive real number:")
+number = float(input())
+frac = str(number).split(".")[1]
+print("The Fractional part of the number is: " + "0."+frac)
+print("THE END of Fractional part problem")
+print("")
```

Comment:
Casting to string, and using `split` is kinda sketchy.
This can be solved without using `str()` or `split()`